### ISSUE TITLE
django_pvstd needs needs to be early in INSTALLED_APPS

### ISSUE THREAD

**Issue by kmpm** [sthima/django-ptvsd#3]
I didn't get the modified runserver to work unless django_pvstd was at top in INSTALLED_APPS.

Probably something about when commands are registerd.

**Comment by Drieger** [comment]
@kmpm, you are right django_ptvsd must be at top of INSTALLED_APPS, or at least before `django.contrib.staticfiles` from where it extends `runserver`. Occording to django documentation thats the way to override a command. 

http://django.readthedocs.io/en/latest/howto/custom-management-commands.html#overriding-commands

Usage should be clear as soon as README is updated on #1.
